core/algorithms/common.py

```diff
@@ -1,19 +1,20 @@
+import geopandas
 import momepy
 from libpysal import graph
 
 
-def continuity(roads):
-    """Assign COINS-based information to roads
+def continuity(roads: geopandas.GeoDataFrame) -> geopandas.GeoDataFrame:
+    """Assign COINS-based information to roads.
 
     Parameters
     ----------
-    roads : GeoDataFrame
-        Road network
+    roads :  geopandas.GeoDataFrame
+        Road network.
 
     Returns
     -------
-    GeoDataFrame
-        original roads with additional columns
+    roads : geopandas.GeoDataFrame
```

Comment:
```suggestion
    geopandas.GeoDataFrame
```

returning objects do not have a name as per numpydoc

Reply:
I [read this](https://numpydoc.readthedocs.io/en/latest/format.html#returns) as optional. Am I missing something? Surely it's not critical if you want it in there, but I like to be as explicit as possible.

Reply:
But that make only sense for tuples, no?

Reply:
I include like the following for nearly every docstring I write (not everything, of course):

```
Returns
--------
<VAR_NAME> : <VAR_TYPE>
    <DESCRIPTION>
```

Reply:
But how do you define what the variable name should be? What is used inside the function is irrelevant. You're trying to _name_ the output? e.g. pandas is not doing that but I see that numpy occasionally does.

Reply:
To be fair, I don't care but I rather have one less thing to think about :)

Reply:
I'm OK to stick closer to `numpy` standards. Shall I update the docstrings I have already written within `core/` in a follow up PR?

***Basic tenet***: omit `<VAR_NAME>` when the returned value(s) are not ambiguous

Reply:
xref #84